candis/app/server/api/data.py

```diff
@@ -231,3 +232,13 @@ def delete():
 
     return json_, code
 
+@app.route(CONFIG.App.Routes.API.Data.DOWNLOAD, methods = ['POST'])
```

Comment:
POST is added as of now- will change it to GET, when download through GEO will be possible.